packages/elina/elina.1.3/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+authors: ["Gagandeep Singh" "Markus Püschel" "Martin Vechev"]
+homepage: "http://elina.ethz.ch/"
+maintainer: "Gagandeep Singh <ggnds@illinois.edu>"
+dev-repo: "git+https://github.com/eth-srl/ELINA.git"
+bug-reports: "https://github.com/eth-srl/ELINA/issues"
+version: "1.3"
+license: "LGPL-3.0-only"
+build: [
+  [
+  "sh"
+  "./configure"
+  "--prefix"
+  "%{share}%/elina"
+  "--apron-prefix"
+  "%{share}%/apron"
+  "--use-opam"
+  "--use-apron"
+  "--use-vector"
+  "--absolute-dylibs" {os = "macos"}
+]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
```

Comment:
This whole `remove` block can be removed since opam 2. Opam is tracking the installed files and knows what to do on uninstall